Cgo errors w/ Go-master (aka Go-1.15)

build fails on Go master (what will be Go-1.15):

```
github.com/ziutek/ftdi
# github.com/ziutek/ftdi
../../../../pkg/mod/github.com/ziutek/ftdi@v0.0.0-20181130113013-aef9e445a2fa/device.go:279:43: cannot use _Ctype_enum_ftdi_interface(c) (type _Ctype_enum_ftdi_interface) as type uint32 in assignment
../../../../pkg/mod/github.com/ziutek/ftdi@v0.0.0-20181130113013-aef9e445a2fa/eeprom.go:194:39: cannot use names[n] (type _Ctype_enum_ftdi_eeprom_value) as type uint32 in assignment
../../../../pkg/mod/github.com/ziutek/ftdi@v0.0.0-20181130113013-aef9e445a2fa/eeprom.go:205:39: cannot use names[n] (type _Ctype_enum_ftdi_eeprom_value) as type uint32 in assignment
../../../../pkg/mod/github.com/ziutek/ftdi@v0.0.0-20181130113013-aef9e445a2fa/eeprom.go:358:39: cannot use cbusFunction[n] (type _Ctype_enum_ftdi_eeprom_value) as type uint32 in assignment
../../../../pkg/mod/github.com/ziutek/ftdi@v0.0.0-20181130113013-aef9e445a2fa/eeprom.go:366:39: cannot use cbusFunction[n] (type _Ctype_enum_ftdi_eeprom_value) as type uint32 in assignment
../../../../pkg/mod/github.com/ziutek/ftdi@v0.0.0-20181130113013-aef9e445a2fa/error.go:24:57: cannot use _Ctype_enum_libusb_error(e) (type _Ctype_enum_libusb_error) as type int32 in argument to _Cfunc_libusb_strerror

```

could this be addressed?
thanks.

--- a/device.go
+++ b/device.go
@@ -276,7 +276,7 @@ func makeDevice(c Channel) (*Device, error) {
 	}
 	d := &Device{ctx}
 	if c != ChannelAny {
-		if e := C.ftdi_set_interface(d.ctx, C.enum_ftdi_interface(c)); e < 0 {
+		if e := C.ftdi_set_interface(d.ctx, uint32(c)); e < 0 {
 			defer d.free()
 			return nil, d.makeError(e)
 		}

--- a/eeprom.go
+++ b/eeprom.go
@@ -191,7 +191,7 @@ func (e EEPROM) channelValue(names []C.enum_ftdi_eeprom_value, c Channel) (
 	if n < 0 || n >= len(names) {
 		panic("bad channel")
 	}
-	C.ftdi_get_eeprom_value(e.d.ctx, names[n], &v)
+	C.ftdi_get_eeprom_value(e.d.ctx, uint32(names[n]), &v)
 	return
 }
 
@@ -202,7 +202,7 @@ func (e EEPROM) setChannelValue(names []C.enum_ftdi_eeprom_value, c Channel,
 	if n < 0 || n >= len(names) {
 		panic("bad channel")
 	}
-	C.ftdi_set_eeprom_value(e.d.ctx, names[n], v)
+	C.ftdi_set_eeprom_value(e.d.ctx, uint32(names[n]), v)
 }
 
 type ChannelType byte
@@ -355,7 +355,7 @@ func (e EEPROM) CBusFunction(n int) CBusFunction {
 		panic("bad CBUS number")
 	}
 	var v C.int
-	C.ftdi_get_eeprom_value(e.d.ctx, cbusFunction[n], &v)
+	C.ftdi_get_eeprom_value(e.d.ctx, uint32(cbusFunction[n]), &v)
 	return CBusFunction(v)
 }
 
@@ -363,7 +363,7 @@ func (e EEPROM) SetCBusFunction(n int, v CBusFunction) {
 	if n < 0 || n >= len(cbusFunction) {
 		panic("bad CBUS number")
 	}
-	C.ftdi_set_eeprom_value(e.d.ctx, cbusFunction[n], C.int(v))
+	C.ftdi_set_eeprom_value(e.d.ctx, uint32(cbusFunction[n]), C.int(v))
 }
 
 func (e EEPROM) Invert() int {

--- a/error.go
+++ b/error.go
@@ -21,5 +21,5 @@ func (e *Error) Error() string {
 type USBError int
 
 func (e USBError) Error() string {
-	return C.GoString(C.libusb_strerror(C.enum_libusb_error(e)))
+	return C.GoString(C.libusb_strerror(int32(e)))
 }

--- a/ftn/device_linux.go
+++ b/ftn/device_linux.go
@@ -104,9 +104,6 @@ func (c *Conn) Serial() (string, error) {
 	return bytes0str(buf), nil
 }
 
-func 
-
-
 /*type Mode byte
 
 const (

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,5 @@
+module github.com/ziutek/ftdi
+
+go 1.13
+
+require github.com/ziutek/lcd v0.0.0-20141212131202-924f223d0903

